ci/record_results.go

```diff
@@ -37,6 +38,17 @@ func (r RunData) RecordResults(logger *zap.SugaredLogger, db database.Database,
 		}
 		logger.Debugf("Updated slip days for %s", r)
 	}
+
+	switch {
```

Comment:
Ref discussion a few minutes ago... Maybe this could be made into a separate helper also on `RunData`. That way, you can test `RecordResults` without requiring a stream service. And you can test `RecordResults` together with a stream service separately (e.g., in a new test).

Reply:
Or to the last point, you can test the stream service without `RecordResults`... by creating the required submission object.